watson/cli.py

```diff
@@ -452,46 +457,62 @@ def report(watson, current, from_, to, projects,
 
     if format_json:
         click.echo(json.dumps(report, indent=4, sort_keys=True))
+        return
+
+    lines = []
+    # use the pager, or print directly to the terminal
+    if pager or (pager is None and
+            watson.config.getboolean('options', 'pager')):
+        use_pager = True
+        def _print(line):
+            lines.append(line)
     else:
-        click.echo('{} -> {}\n'.format(
-            style('date', '{:ddd DD MMMM YYYY}'.format(
-                arrow.get(report['timespan']['from'])
+        use_pager = False
+        def _print(line):
+            click.echo(line)
+
+    _print('{} -> {}\n'.format(
+        style('date', '{:ddd DD MMMM YYYY}'.format(
+            arrow.get(report['timespan']['from'])
+        )),
+        style('date', '{:ddd DD MMMM YYYY}'.format(
+            arrow.get(report['timespan']['to'])
+        ))
+        ))
```

Comment:
Line should be on the same indentation level as `_print`.

Reply:
fixed!